packages/rocketchat-integrations/server/api/api.js

```diff
@@ -0,0 +1,358 @@
+/* globals Api Meteor Restivus logger processWebhookMessage*/
+// TODO: remove globals
+import vm from 'vm';
+import moment from 'moment';
+
+const compiledScripts = {};
+
+const buildSandbox = function(store = {}) {
+	const sandbox = {
+		_,
+		s,
+		console,
+		moment,
+		Store: {
+			set(key, val) {
+				return store[key] = val;
+			},
+			get(key) {
+				return store[key];
+			}
+		},
+		HTTP(method, url, options) {
+			try {
+				return {
+					result: HTTP.call(method, url, options)
+				};
+			} catch (error) {
+				return {
+					error
+				};
+			}
+		}
+	};
+
+	Object.keys(RocketChat.models).filter(function(k) {
+		return !k.startsWith('_');
+	}).forEach((k) => {
+		return sandbox[k] = RocketChat.models[k];
+	});
+	return { store, sandbox	};
+};
+
+const getIntegrationScript = function(integration) {
+	const compiledScript = compiledScripts[integration._id];
+	if ((compiledScript != null) && +compiledScript._updatedAt === +integration._updatedAt) {
+		return compiledScript.script;
+	}
+	const script = integration.scriptCompiled;
+	const sandboxItems = buildSandbox();
```

Comment:
Break this out using the spread operator? `const { store, sandbox } = buildSandbox();` And then use it below, as it'd clean this up and make it very similar to how the outgoing webhook script looks as well.